### DISCUSSION TITLE
Low INT?

### DISCUSSION THREAD

**Discussion by rubyFeedback** [Spellhold-Studios/Tashia-NPC#1] [Suggestions]
One thing that slightly troubles me about the NPC is the low INT of 15.

There are means to increase it in the game, with other mods, e. g. Lava's helmet that increased INT by +2 and other things. But even then I think an INT of 15 is a bit too low for the primary attribute. Could it be increased to 16? For comparison, Aerie has an INT of 16 and is dual-class:

https://baldursgate.fandom.com/wiki/Aerie

Jan has an INT of 16 and is dual class:

https://baldursgate.fandom.com/wiki/Jan_Jansen

Edwin has an INT of 18 and is primary mage class:

https://baldursgate.fandom.com/wiki/Edwin_Odesseiron

Neera has an INT of 17 and is primary mage class:

https://baldursgate.fandom.com/wiki/Neera

Imoen has an INT of 17 and is dual class:

https://baldursgate.fandom.com/wiki/Imoen

Nalia has an INT of 17 and is dual class:

https://baldursgate.fandom.com/wiki/Nalia_de%27Arnise

**Comment by Argent77** [comment]
Sorcerers don't have primary attributes. It would make sense to make a sorcerer not too stupid for roleplaying purposes (and to use mage-specific items), but INT is not required for any of their special abilities (e.g. scribing spells).

In 3e the primary attribute for sorcerers to be able to cast spells would be CHA, but that hasn't been implemented in the game engine either.
